content/tugboat-cli/install-the-cli.md

```diff
@@ -4,8 +4,10 @@ date: 2019-09-19T10:41:37-04:00
 weight: 1
 ---
 
-The Tugboat Command Line Tool is available for Windows, MacOS, and Linux. In all three cases, it is a stand-alone
-binary.
+The Tugboat Command Line Tool is available for Windows, MacOS, and Linux. In all three cases, we offer a stand-alone
+binary. MacOS users can also [install and manage the Tugboat CLI via Homebrew](#homebrew).
+
+## To use the stand-alone binary:
```

Comment:
Now that we are recommending homebrew for macOS, this workflow is no longer universal. Let's remove these steps from the top, and include them in each section that they apply to individually. That also lets each set of per-OS instructions stand on their own.